hudi-common/src/main/avro/HoodieClusteringGroup.avsc

```diff
@@ -40,6 +40,11 @@
          }],
          "default": null
       },
+      {
+         "name":"numOutputGroups",
```

Comment:
should it be called `numOutputSlices`?  the whole thing  is called a group right?  or may be you mean `numOutputFileGroups`? in which case, lets rename this to clarify

Reply:
Changed it to numOutputFileGroups